public/setup/atomicupdate/updatepath.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$sql[]="UPDATE `{$dbprefix}acces`SET `page` = '' WHERE `page`='personnel/index.php';";
```

Comment:
Juste remove the page from this entry

Reply:
I do not understand your point, could you please explain it to me ?